src/mumble/TalkingUI.cpp

```diff
@@ -569,6 +571,41 @@ void TalkingUI::mousePressEvent(QMouseEvent *event) {
 	updateUI();
 }
 
+void TalkingUI::wheelEvent(QWheelEvent *event) {
+	if (!event->modifiers().testFlag(Qt::ControlModifier)) {
+		return QWidget::wheelEvent(event);
+	}
+
+	if (!Global::get().s.talkingUI_CtrlScrollLocalVolAdj) {
+		return;
+	}
+
+	ClientUser *user = Global::get().mw->pmModel->getSelectedUser();
```

Comment:
"Weird" things happen, if you enable to always show your local user (or channel listeners) and use this scroll event. Mainly, it changes the local volume of yourself which has currently no effect IIRC, but can not be changed back using the "self" context menu.

Unless we plan to use self-local-volume-adjustment for something, I think you should add a check that we are not able to change the local volume of ourselves.